<!DOCTYPE html>



  


<html class="theme-next muse use-motion" lang="">
<head>
  <meta charset="UTF-8"/>
<meta http-equiv="X-UA-Compatible" content="IE=edge" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2"/>
<meta name="theme-color" content="#222">












<meta http-equiv="Cache-Control" content="no-transform" />
<meta http-equiv="Cache-Control" content="no-siteapp" />






















<link href="/lib/font-awesome/css/font-awesome.min.css?v=4.6.2" rel="stylesheet" type="text/css" />

<link href="/css/main.css?v=6.1.0" rel="stylesheet" type="text/css" />


  <link rel="apple-touch-icon" sizes="180x180" href="/images/apple-touch-icon-next.png?v=6.1.0">


  <link rel="icon" type="image/png" sizes="32x32" href="/images/favicon-32x32-next.png?v=6.1.0">


  <link rel="icon" type="image/png" sizes="16x16" href="/images/favicon-16x16-next.png?v=6.1.0">


  <link rel="mask-icon" href="/images/logo.svg?v=6.1.0" color="#222">









<script type="text/javascript" id="hexo.configurations">
  var NexT = window.NexT || {};
  var CONFIG = {
    root: '/',
    scheme: 'Muse',
    version: '6.1.0',
    sidebar: {"position":"left","display":"post","offset":12,"b2t":false,"scrollpercent":false,"onmobile":false},
    fancybox: false,
    fastclick: false,
    lazyload: false,
    tabs: true,
    motion: {"enable":true,"async":false,"transition":{"post_block":"fadeIn","post_header":"slideDownIn","post_body":"slideDownIn","coll_header":"slideLeftIn","sidebar":"slideUpIn"}},
    algolia: {
      applicationID: '',
      apiKey: '',
      indexName: '',
      hits: {"per_page":10},
      labels: {"input_placeholder":"Search for Posts","hits_empty":"We didn't find any results for the search: ${query}","hits_stats":"${hits} results found in ${time} ms"}
    }
  };
</script>


  




  <meta name="description" content="AnnoVar 结果的注释 Chr: 染色体号 start：基因起始位点  end：基因终止位点 Ref: hg19, 该位点参考基因 Alt: 该位点突变基因，即突变结果 Func.refGene：基因所在功能区，例如exonic，intronic，UTR3，UTR5，splicing，upstream，downstream，intergenic。此处的exonic特指外显子编码氨基酸区，不包括">
<meta property="og:type" content="article">
<meta property="og:title" content="AnnoVar snpEff VEP 结果注释">
<meta property="og:url" content="http://yoursite.com/2018/05/21/AnnoVar_snpEff_VEP_result_anno/index.html">
<meta property="og:site_name" content="Viva La Vida">
<meta property="og:description" content="AnnoVar 结果的注释 Chr: 染色体号 start：基因起始位点  end：基因终止位点 Ref: hg19, 该位点参考基因 Alt: 该位点突变基因，即突变结果 Func.refGene：基因所在功能区，例如exonic，intronic，UTR3，UTR5，splicing，upstream，downstream，intergenic。此处的exonic特指外显子编码氨基酸区，不包括">
<meta property="og:locale" content="default">
<meta property="og:updated_time" content="2018-05-21T07:59:43.000Z">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="AnnoVar snpEff VEP 结果注释">
<meta name="twitter:description" content="AnnoVar 结果的注释 Chr: 染色体号 start：基因起始位点  end：基因终止位点 Ref: hg19, 该位点参考基因 Alt: 该位点突变基因，即突变结果 Func.refGene：基因所在功能区，例如exonic，intronic，UTR3，UTR5，splicing，upstream，downstream，intergenic。此处的exonic特指外显子编码氨基酸区，不包括">






  <link rel="canonical" href="http://yoursite.com/2018/05/21/AnnoVar_snpEff_VEP_result_anno/"/>



<script type="text/javascript" id="page.configurations">
  CONFIG.page = {
    sidebar: "",
  };
</script>

  <title>AnnoVar snpEff VEP 结果注释 | Viva La Vida</title>
  









  <noscript>
  <style type="text/css">
    .use-motion .motion-element,
    .use-motion .brand,
    .use-motion .menu-item,
    .sidebar-inner,
    .use-motion .post-block,
    .use-motion .pagination,
    .use-motion .comments,
    .use-motion .post-header,
    .use-motion .post-body,
    .use-motion .collection-title { opacity: initial; }

    .use-motion .logo,
    .use-motion .site-title,
    .use-motion .site-subtitle {
      opacity: initial;
      top: initial;
    }

    .use-motion {
      .logo-line-before i { left: initial; }
      .logo-line-after i { right: initial; }
    }
  </style>
</noscript>

</head>

<body itemscope itemtype="http://schema.org/WebPage" lang="default">

  
  
    
  

  <div class="container sidebar-position-left page-post-detail">
    <div class="headband"></div>

    <header id="header" class="header" itemscope itemtype="http://schema.org/WPHeader">
      <div class="header-inner"><div class="site-brand-wrapper">
  <div class="site-meta ">
    

    <div class="custom-logo-site-title">
      <a href="/" class="brand" rel="start">
        <span class="logo-line-before"><i></i></span>
        <span class="site-title">Viva La Vida</span>
        <span class="logo-line-after"><i></i></span>
      </a>
    </div>
      
        <p class="site-subtitle"></p>
      
  </div>

  <div class="site-nav-toggle">
    <button aria-label="Toggle navigation bar">
      <span class="btn-bar"></span>
      <span class="btn-bar"></span>
      <span class="btn-bar"></span>
    </button>
  </div>
</div>




<nav class="site-nav">
  
    <ul id="menu" class="menu">
      
        
        
          
  
  <li class="menu-item menu-item-home">
    <a href="/" rel="section">
      <i class="menu-item-icon fa fa-fw fa-home"></i> <br />Home</a>
</li>

      
        
        
          
  
  <li class="menu-item menu-item-archives">
    <a href="/archives/" rel="section">
      <i class="menu-item-icon fa fa-fw fa-archive"></i> <br />Archives</a>
</li>

      

      
      
    </ul>
  

  
    

  

  
</nav>



  



</div>
    </header>

    


    <main id="main" class="main">
      <div class="main-inner">
        <div class="content-wrap">
          
          <div id="content" class="content">
            

  <div id="posts" class="posts-expand">
    

  

  
  
  

  

  <article class="post post-type-normal" itemscope itemtype="http://schema.org/Article">
  
  
  
  <div class="post-block">
    <link itemprop="mainEntityOfPage" href="http://yoursite.com/2018/05/21/AnnoVar_snpEff_VEP_result_anno/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="name" content="Leah Sunny">
      <meta itemprop="description" content="">
      <meta itemprop="image" content="/images/avatar.gif">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="Viva La Vida">
    </span>

    
      <header class="post-header">

        
        
          <h1 class="post-title" itemprop="name headline">AnnoVar snpEff VEP 结果注释</h1>
        

        <div class="post-meta">
          <span class="post-time">
            
              <span class="post-meta-item-icon">
                <i class="fa fa-calendar-o"></i>
              </span>
              
                <span class="post-meta-item-text">Posted on</span>
              
              <time title="Post created" itemprop="dateCreated datePublished" datetime="2018-05-21T15:34:34+08:00">2018-05-21</time>
            

            
            

            
          </span>

          

          
            
          

          
          

          

          

          

        </div>
      </header>
    

    
    
    
    <div class="post-body" itemprop="articleBody">

      
      

      
        <h2 id="AnnoVar-结果的注释"><a href="#AnnoVar-结果的注释" class="headerlink" title="AnnoVar 结果的注释"></a>AnnoVar 结果的注释</h2><ul>
<li>Chr: 染色体号</li>
<li>start：基因起始位点 </li>
<li>end：基因终止位点</li>
<li>Ref: hg19, 该位点参考基因</li>
<li>Alt: 该位点突变基因，即突变结果</li>
<li>Func.refGene：基因所在功能区，例如exonic，intronic，UTR3，UTR5，splicing，upstream，downstream，intergenic。此处的exonic特指外显子编码氨基酸区，不包括外显子的UTR区</li>
<li>Gene.refGene：基因名称</li>
<li>GeneDetail.refGene：位点突变的具体信息，比如：NM_024408:c.-148G&gt;A,NM_001200001:c.-148G&gt;A* ExonicFunc.refGene：外显子区功能改变</li>
<li>AAChange.refGene：氨基酸改变，例如ARID1A:NM_006015:exon20:c.G5542T:p.G1848W,ARID1A:NM_139135:exon20:c.G4891T:p.G1631W</li>
<li>cytoBand：cytoBand中的注释信息，IGV也会给出该信息，3p2,1q12, 染色体号，p断臂，q长臂，band条带</li>
<li>cosmic81：cosmic中的注释信息，该变异在癌症体细胞突变数据库中的cosmicID</li>
<li>CLINSIG：该变异在ClinVar数据库中的临床意义（Benign，Likelybenign，Uncertainsignificance，Likelypathogenic，Pathogenic，Drug-response）</li>
<li>CLNDBN：该变异所引起的疾病名称</li>
<li>CLNACC：变异在CLINVAR数据库中的accession号和版本号， 例如：RCV000161067.2|RCV000168150.1</li>
<li>CLNDSDB：疾病关联信息的数据库来源</li>
<li>CLNDSDBID：该变异所引起疾病所在数据库中的ID</li>
<li>esp6500siv2_all：国家心肺和血液研究所外显子组测序计划，包含SNP变异、InDel变异和Y染色体上的变异的所有个体中，突变碱基的等位基因频率（alternative allele frequency）</li>
<li>avsnp147：avsnp147 中的注释信息,dbSNP version 147 with left-normalization ， 例如：rs730882025</li>
</ul>
<h2 id="snpEff-结果的注释"><a href="#snpEff-结果的注释" class="headerlink" title="snpEff 结果的注释"></a>snpEff 结果的注释</h2><ul>
<li>Chr: 染色体号</li>
<li>Pos: 染色体位置，Ref:参考基因， Alt:突变结果</li>
<li>Vcf_Format： vcf 结果形式，GT:AD:BQ:DP:FA:SS<ul>
<li>GT: 基因型（Genotype）, 0代表allele和ref相同， 1，2，3代表allele和ref不同， 0/0,1/1代表纯合，0/1代表杂合</li>
<li>AD: 参考基因的个数，突变基因的个数</li>
<li>BQ: 参考基因的的碱基质量</li>
<li>DP: 该位点覆盖的总reads 数 （就是AD那列， 两个数相加）</li>
<li>FA: 突变基因的百分比（AD中突变基因的个数 / DP)</li>
<li>SS: 突变状态的表示，0代表wildtype, 1代表germline, 2代表somatic, 3代表LOH，4代表post-transcriptional modification, 5代表unknown</li>
</ul>
</li>
<li>Vcf_Format_Info：数据的vcf 结果， 例如：0:21,1:.:22:0.045:0，</li>
<li>Alt_Percent： 突变基因百分比，就是从Vcf_Format_Info中提取的 FA 那列，就是 0.045</li>
<li>Depth：该位点覆盖的总reads数量，就是DP 值， 是上述例子中的，22</li>
<li>Alt_Depth： 突变基因的个数，就是从AD中提取的逗号后面的数值</li>
<li>HGVS.c：Human Genome Variation Society.condons, 碱基的改变，<mark>写法是c.953G&gt;A</mark></li>
<li>cDNA.pos/cDNA.length： cDNA 的位置，以及总长度， 例如406/2581， </li>
<li>CDS.pos/CDS.length： CDS (编码区）的位置，以及总长度，例如 367/858</li>
<li>HGVS.p：(Human Genome Variation Society.protein) 氨基酸的改变，<mark>写法是p.R318Q</mark></li>
<li>AA.pos/AA.length：突变氨基酸的位置，以及该基因的总氨基酸的个数， 例如318/585</li>
<li>Feature_Type： 功能区域， 例如： transcript,  cross_link</li>
<li>Feature_ID：Ensembl 中transcript的id号， 例如 ENST00000359008</li>
<li>Transcript_BioType：突变基因转录后的功能，一般是protein_coding</li>
<li>Rank：表示该突变所在的外显子号，以及该基因一共有几个外显子 19/28(EGFR)</li>
</ul>
<h2 id="VEP结果的注释"><a href="#VEP结果的注释" class="headerlink" title="VEP结果的注释"></a>VEP结果的注释</h2><ul>
<li>CHROM: 染色体号</li>
<li>POS: 位置信息，</li>
<li>REF: 参考基因，例如 C</li>
<li>ALT: 突变基因，例如A</li>
<li><p>INFO :  这个是注释的主体，分为以下两部分  </p>
<blockquote>
</blockquote>
<p>  数据整体状况的总结 ，例子：DP=381;ECNT=1;NLOD=27.00;N_ART_LOD=-1.963e+00;POP_AF=1.000e-03;P_GERMLINE=-2.369e+01;TLOD=20.74;</p>
<ul>
<li>DP : Approximate read depth; some reads may have been filtered， 该位点reads 的覆盖度，reads 是经过筛选的</li>
<li>ECNT : Number of events in this haplotype，</li>
<li>NLOD : Normal LOD score，该位点在正常细胞中连锁的概率</li>
<li>N_ART_LOD : log odds of artifact in normal with same allele fraction as tumor</li>
<li>POP_AF : population allele frequencies of alt alleles，</li>
<li>P_GERMLINE : Posterior probability for alt allele to be germline variants</li>
<li><p>TLOD : Tumor LOD score</p>
<blockquote>
<p>来自Ensemble的注释结果部分， CSQ 部分</p>
</blockquote>
</li>
<li><p>Allele : 突变基因， 对应着之前的 ALT</p>
</li>
<li>consequence:  突变类型</li>
<li>impact:  对突变的影响的预测, 分为HIGH, MODERATE,LOW,MODIFIER四类</li>
<li>symbol : 基因名称</li>
<li>gene：ensemble 中 的gene-id, 例如ENSG00000004455</li>
<li>Feature_type： 功能区域， 例如： transcript,  cross_link</li>
<li>feature: Ensembl 中transcript的id号， 例如 ENST00000354858</li>
<li>biotype :  功能注释，例如protein_coding</li>
<li>exon: 外显子号/该基因的外显子总数，19/28(EGFR)</li>
<li>intron : 内含子号/该基因的内含子总数 </li>
<li>HGVSc: Human Genome Variation Society.condons, 碱基的改变，例如ENST00000354858.6:c.614G&gt;A</li>
<li>HGVSp： (Human Genome Variation Society.protein) 氨基酸的改变，例如ENSP00000346921.6:p.Gly205Glu</li>
<li>cDNA_position：cDNA的位置</li>
<li>CDS_position： CDS 的位置</li>
<li>Protein_position： 蛋白的位置</li>
<li>Amino_acids： 氨基酸的改变， 简写， 例如G/E</li>
<li>Codons： 碱基的改变，对应着上面的氨基酸的改变， 大写的为改变的碱基，例如gGg/gAg</li>
<li>Existing_variation ：COSMIC=”81” ClinVar=”201706”，中的注释结果，例如rs202182972&amp;COSM1560031</li>
<li>distance: 距离转录本最短的距离</li>
<li>strand: 突变的正负链，正链是1，负链是 -1</li>
<li>variant_class: 突变类型， 例如 SNV</li>
<li>SYMBOL_SOURCE：数据来源，HGNC, 这是固定的</li>
<li>HGNC_ID : 例子是362， HGNC is responsible for approving unique symbols and names for human loci, including protein coding genes, ncRNA genes and pseudogenes, to allow unambiguous scientific communication, </li>
<li>canonical:  经典转录本的表示，例如：yes,  判断的标准为：For human, the canonical transcript for a gene is set according to the following hierarchy: 1. Longest CCDS translation with no stop codons. 2. If no (1), choose the longest Ensembl/Havana merged translation with no stop codons. 3. If no (2), choose the longest translation with no stop codons. 4. If no translation, choose the longest non-protein-coding transcript，</li>
<li>TSL：Adds the transcript support level for this transcript to the output. <a>NB: not available for GRCh37.</a>The Transcript Support Level (TSL) is a method to highlight the well-supported and poorly-supported transcript models for users. The method relies on the primary data that can support full-length transcript structure: mRNA and EST alignments supplied by UCSC and Ensembl。</li>
<li>APPRIS：Adds the APPRIS isoform annotation for this transcript to the output. <a>NB: not available for GRCh37.Not used by default.</a> APPRIS is a system to annotate alternatively spliced transcripts based on a range of computational methods. It provides value to the annotations of the human, mouse, zebrafish, rat, and pig genomes. </li>
<li>CCDS: 例如：CCDS374.1，Adds the CCDS transcript identifer (where available) to the output.The CCDS project is a collaborative effort to identify a core set of protein coding regions that are consistently annotated and of high quality.</li>
<li>ENSP： Add the Ensembl protein identifier to the output where appropriate. 例如：ENSP00000346921，与之前的HGVSp一样，只是版本号的控制没有那么严格。</li>
<li>SWISSPROT ： Uniprot 中，SWISSPROT注释库的编号，例如：P54819</li>
<li>TREMBL：例如：F8W1A4。 Adds best match accessions for translated protein products from three UniProt-related databases (SWISSPROT, TREMBL and UniParc) to the output. Not used by default. <a><a href="https://www.ebi.ac.uk/uniprot" target="_blank" rel="noopener">https://www.ebi.ac.uk/uniprot</a> </a></li>
<li>UNIPARC：同上，来源与Uniport 的蛋白注释库。例如: UPI000016781E</li>
<li>RefSeq:  依照RefSeq cache 显示的，该位点对应的参考基因组的位置信息。例如refseq 编号为 refseq=”01_2015” 。NM_001625.3</li>
<li>REFSEQ_MATCH: For the GRCh38 VEP cache, each RefSeq transcript is annotated with the REFSEQ_MATCH flag indicating whether and how the RefSeq model differs from the underlying genome. Note that currently the REFSEQ_MATCH flag will not be set when using the GRCh37 cache.</li>
<li>SOURCE ： 注释来源，比如Ensembl。Use the merged Ensembl and RefSeq cache. Consequences are flagged with the SOURCE of each transcript used.</li>
<li>GENE_PHENO : Indicates if the overlapped gene is associated with a phenotype, disease or trait。</li>
<li>sift: 预测一个氨基酸的改变是否影响蛋白的功能。例如deleterious(0.02)。评判标准为：Positions with normalized probabilities less than 0.05 are predicted to be deleterious, those greater than or equal to 0.05 are predicted to be tolerated. Species limited SIFT predicts whether an amino acid substitution affects protein function based on sequence homology and the physical properties of amino acids. <a><a href="http://sift.jcvi.org/www/SIFT_help.html#SIFT_PROCEDURE" target="_blank" rel="noopener">http://sift.jcvi.org/www/SIFT_help.html#SIFT_PROCEDURE</a></a></li>
<li>polyphen:     跟上面一个作用，预测氨基酸的改变是否影响蛋白的功能。例如：possibly_damaging(0.905)。Human only PolyPhen is a tool which predicts possible impact of an amino acid substitution on the structure and function of a human protein using straightforward physical and comparative considerations.</li>
<li>DOMAINS ： Adds names of overlapping protein domains to output。因为蛋白是三维的，一个蛋白会有多个domain组成，所以这里标注了重叠的的domains名称。例如：HAMAP:MF_00235&amp;HAMAP:MF_03168&amp;hmmpanther:PTHR23359&amp;hmmpanther:PTHR23359:SF69&amp;Gene3D:3.40.50.300</li>
<li><p>HGVS_OFFSET：human genome variation society, <a><a href="https://varnomen.hgvs.org" target="_blank" rel="noopener">https://varnomen.hgvs.org</a></a>    </p>
<blockquote>
<blockquote>
</blockquote>
<p>以下是ExAC的数据库注释结果，ExAC是Exome Aggregation Consortium的简称，整合了60706个无亲缘关系个体的数据，这些个体来源于大量disease-specific研究和群体遗传学研究，能够用做严重疾病研究的reference set of allele frequency。目前ExAC数据库中包括ALL, AFR (African), AMR (Admixed American), EAS (East Asian), FIN (Finnish), NFE (Non-finnish European), OTH (other), SAS (South Asian)。ExAC_ALL是指在所有人群中，该变异位点上突变碱基的等位基因频率。</p>
</blockquote>
<blockquote>
<blockquote>
</blockquote>
<p><a><a href="http://gnomad.broadinstitute.org" target="_blank" rel="noopener">http://gnomad.broadinstitute.org</a></a>.  Include allele frequency from Genome Aggregation Database (gnomAD) exome populations. Note only data from the gnomAD exomes are included.The Genome Aggregation Database (gnomAD) is a resource developed by an international coalition of investigators, with the goal of aggregating and harmonizing both exome and genome sequencing data from a wide variety of large-scale sequencing projects, and making summary data available for the wider scientific community.<br>The data set provided on this website spans 123,136 exome sequences and 15,496 whole-genome sequences from unrelated individuals sequenced as part of various disease-specific and population genetic studies.<br>Genome Aggregation Database(简称gnomAD)是由各国研究者联合发展起来的基因组突变频率数据库。其目的是汇集和协调来自众多大规模测序计划的全外显子组和全基因组测序数据，为广泛的科学研究团体汇总数据。<br>该数据库提供的数据集包括123,136个个体的全外显子组测序数据和15,496个个体的全基因组测序数据，这些数据来源于各种疾病研究项目及大型人群测序项目。</p>
<blockquote>
</blockquote>
</blockquote>
</li>
<li>gnomAD_AF：该突变在gnomeAD 中的突变频率</li>
<li>gnomAD_AFR_AF：非洲人中的突变频率</li>
<li>gnomAD_AMR_AF：混血美国人中的突变频率</li>
<li>gnomAD_ASJ_AF：德系犹太人</li>
<li>gnomAD_EAS_AF：在东亚人中的突变频率</li>
<li>gnomAD_FIN_AF：芬兰人</li>
<li>gnomAD_NFE_AF：非芬兰欧洲人。</li>
<li>gnomAD_OTH_AF：在其他人</li>
<li>gnomAD_SAS_AF：在南亚人中的突变频率</li>
<li>MAX_AF：给出在以上各个人群中，最高的突变率是多少</li>
<li>MAX_AF_POPS：给出在以上各个人群中，突变频率最高的人群名称<blockquote>
<blockquote>
</blockquote>
</blockquote>
</li>
</ul>
</li>
</ul>
<p>Checks for the existence of known variants that are co-located with your input.与已知数据进行对比注释    - CLIN_SIG:</p>
<pre><code>- SOMATIC：
- PHENO：
- PUBMED：提供与此突变相关的pubmed ID, 但是默认是没有的，需要自己添加cache。Report Pubmed IDs for publications that cite existing variant. Must be used with --cache.  
</code></pre><p>The script can also call if a variant falls in a high information position within a transcription factor binding site。默认没有。motif 是保守的一段氨基酸序列，也表示一种超二级结构，常见的motif 为转录因子中的DNA结区。</p>
<pre><code>- MOTIF\_NAME：
- MOTIF\_POS：
- HIGH\_INF\_POS：
- MOTIF\_SCORE\_CHANGE：
</code></pre><figure class="highlight bash"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br></pre></td><td class="code"><pre><span class="line">VEP官网，非常的详细</span><br><span class="line">https://uswest.ensembl.org/info/docs/tools/vep/script/vep_options.html</span><br></pre></td></tr></table></figure>
<ul>
<li>FORMAT: 就是上一步call出来的vcf 信息，例如GT:AD:AF:F1R2:F2R1:MBQ:MFRL:MMQ:MPOS:SA_MAP_AF:SA_POST_PROB, 这是Mutect2 的vcf 结果文件<ul>
<li>GT: 基因型（Genotype）, 0代表allele和ref相同， 1，2，3代表allele和ref不同， 0/0,1/1代表纯合，0/1代表杂合</li>
<li>AD: 参考基因的个数，突变基因的个数。例如：77，16</li>
<li>AF: 突变率， 突变基因的个数 / （突变基因的个数+ 参考基因的个数 ）</li>
<li>F1R2: Count of reads in F1R2 pair orientation supporting each allele。 以F1R2的形式匹配的reads 数量。形式为：参考基因个数，突变基因个数。例如：40，12</li>
<li>F2R1: 以F2R1的形式匹配的reads 数量。形式为：参考基因个数，突变基因个数。例如：37,4</li>
<li>MBQ:  碱基质量的中位数，例如：32</li>
<li>MFRL:  fragment 长度的中位数， 例如：177，184</li>
<li>MMQ: mapping 质量的中位数， 例如：60</li>
<li>MPOS: 突变位点距离reads 末端距离的中位数，例如：20</li>
<li>SA_MAP_AF：MAP estimates of allele fraction given z</li>
<li>SA_POST_PROB：posterior probabilities of the presence of strand artifact</li>
</ul>
</li>
</ul>
<h3 id="备注"><a href="#备注" class="headerlink" title="备注"></a>备注</h3><blockquote>
<p>clinvar：注释变异与人类健康之间的关系，临床意义的数据来源于NCBI 格式为：CLINSIG=non-pathogenic;CLNDBN=not_specified;CLNREVSTAT=single;CLNACC=RCV000116259.1;CLNDSDB=MedGen;CLNDSDBID=CN169374。CLINSIG代表变异位点在临床意义，可取值为unknown, untested, non-pathogenic, probable-non-pathogenic, probable-pathogenic, pathogenic, drug-response, histocompatibility, other。<br>CLINDBN代表变异位点相关的疾病名称。CLNREVSTAT 代表该条临床信息的核查情况，取值为mult、single、not、exp、prof。CLNACC代表变异在CLINVAR数据库中的accession号和版本号。CLNDSDB是疾病关联信息的数据库来源，CLNDSDBID是数据库中的编号。</p>
</blockquote>
<p><mark>Putative_impact</mark>: 是对突变影响的预测，共有4个程度, <a><a href="https://documentation.partek.com/pages/viewpage.action?pageId=6717754" target="_blank" rel="noopener">https://documentation.partek.com/pages/viewpage.action?pageId=6717754</a></a></p>
<blockquote>
<blockquote>
<blockquote>
<ul>
<li>HIGH - frame shifts, addition/deletion of stop codons, etc; </li>
<li><mark>MODERATE – codon change/deletion/insertion, etc;</mark> </li>
<li>LOW – synonymous changes, etc; </li>
<li>MODIFIER – changes outside coding regions, etc.</li>
</ul>
</blockquote>
</blockquote>
</blockquote>
<blockquote>
</blockquote>
<p>How is the LOD score calculated?<br>The LOD score is calculated as follows<br>LOD = Z = log 10 (probability of birth sequence with a given linkage value/ probability of birth sequence with no linkage)<br>As a general rule, if the LOD score is 3 or higher, it means that there is a high probability of genetic linkage. Lower number indicates less likelihood of genetic linkage, even though they can still be useful in the process of elimination. A LOD score of +3 indicates 1000 to 1 odds that the linkage being observed did not occur by chance.</p>

      
    </div>

    

    
    
    

    

    

    

    <footer class="post-footer">
      

      
      
      

      
        <div class="post-nav">
          <div class="post-nav-next post-nav-item">
            
              <a href="/2018/05/08/python3网络爬虫实战_安装/" rel="next" title="python3网络爬虫实战--安装">
                <i class="fa fa-chevron-left"></i> python3网络爬虫实战--安装
              </a>
            
          </div>

          <span class="post-nav-divider"></span>

          <div class="post-nav-prev post-nav-item">
            
          </div>
        </div>
      

      
      
    </footer>
  </div>
  
  
  
  </article>



    <div class="post-spread">
      
    </div>
  </div>


          </div>
          

  



        </div>
        
          
  
  <div class="sidebar-toggle">
    <div class="sidebar-toggle-line-wrap">
      <span class="sidebar-toggle-line sidebar-toggle-line-first"></span>
      <span class="sidebar-toggle-line sidebar-toggle-line-middle"></span>
      <span class="sidebar-toggle-line sidebar-toggle-line-last"></span>
    </div>
  </div>

  <aside id="sidebar" class="sidebar">
    
    <div class="sidebar-inner">

      

      
        <ul class="sidebar-nav motion-element">
          <li class="sidebar-nav-toc sidebar-nav-active" data-target="post-toc-wrap">
            Table of Contents
          </li>
          <li class="sidebar-nav-overview" data-target="site-overview-wrap">
            Overview
          </li>
        </ul>
      

      <section class="site-overview-wrap sidebar-panel">
        <div class="site-overview">
          <div class="site-author motion-element" itemprop="author" itemscope itemtype="http://schema.org/Person">
            
              <p class="site-author-name" itemprop="name">Leah Sunny</p>
              <p class="site-description motion-element" itemprop="description"></p>
          </div>

          
            <nav class="site-state motion-element">
              
                <div class="site-state-item site-state-posts">
                
                  <a href="/archives/">
                
                    <span class="site-state-item-count">4</span>
                    <span class="site-state-item-name">posts</span>
                  </a>
                </div>
              

              

              
            </nav>
          

          

          

          
          

          
          

          
            
          
          

        </div>
      </section>

      
      <!--noindex-->
        <section class="post-toc-wrap motion-element sidebar-panel sidebar-panel-active">
          <div class="post-toc">

            
              
            

            
              <div class="post-toc-content"><ol class="nav"><li class="nav-item nav-level-2"><a class="nav-link" href="#AnnoVar-结果的注释"><span class="nav-number">1.</span> <span class="nav-text">AnnoVar 结果的注释</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#snpEff-结果的注释"><span class="nav-number">2.</span> <span class="nav-text">snpEff 结果的注释</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#VEP结果的注释"><span class="nav-number">3.</span> <span class="nav-text">VEP结果的注释</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#备注"><span class="nav-number">3.1.</span> <span class="nav-text">备注</span></a></li></ol></li></ol></div>
            

          </div>
        </section>
      <!--/noindex-->
      

      

    </div>
  </aside>


        
      </div>
    </main>

    <footer id="footer" class="footer">
      <div class="footer-inner">
        <div class="copyright">&copy; <span itemprop="copyrightYear">2018</span>
  <span class="with-love" id="animate">
    <i class="fa fa-user"></i>
  </span>
  <span class="author" itemprop="copyrightHolder">Leah Sunny</span>

  

  
</div>




  <div class="powered-by">Powered by <a class="theme-link" target="_blank" href="https://hexo.io">Hexo</a> v3.7.1</div>



  <span class="post-meta-divider">|</span>



  <div class="theme-info">Theme &mdash; <a class="theme-link" target="_blank" href="https://github.com/theme-next/hexo-theme-next">NexT.Muse</a> v6.1.0</div>




        








        
      </div>
    </footer>

    
      <div class="back-to-top">
        <i class="fa fa-arrow-up"></i>
        
      </div>
    

    

  </div>

  

<script type="text/javascript">
  if (Object.prototype.toString.call(window.Promise) !== '[object Function]') {
    window.Promise = null;
  }
</script>


























  
  
    <script type="text/javascript" src="/lib/jquery/index.js?v=2.1.3"></script>
  

  
  
    <script type="text/javascript" src="/lib/velocity/velocity.min.js?v=1.2.1"></script>
  

  
  
    <script type="text/javascript" src="/lib/velocity/velocity.ui.min.js?v=1.2.1"></script>
  


  


  <script type="text/javascript" src="/js/src/utils.js?v=6.1.0"></script>

  <script type="text/javascript" src="/js/src/motion.js?v=6.1.0"></script>



  
  

  
  <script type="text/javascript" src="/js/src/scrollspy.js?v=6.1.0"></script>
<script type="text/javascript" src="/js/src/post-details.js?v=6.1.0"></script>



  


  <script type="text/javascript" src="/js/src/bootstrap.js?v=6.1.0"></script>



  



	





  





  










  





  

  

  

  

  
  

  

  

  

  

</body>
</html>
